src/plugins/explore/public/components/query_panel/query_panel_widgets/query_panel_widgets.tsx

```diff
@@ -32,14 +24,6 @@ export const QueryPanelWidgets = () => {
         {/* TODO: Actions should go here */}
         <QueryPanelError />
       </div>
-
-      {/* Right Section */}
-      <div className="exploreQueryPanelWidgets__right">
```

Comment:
If you remove this div (which you shouldn't) then you should also erase the styling in the scss file